config.example.yaml

```diff
@@ -61,50 +61,50 @@ db:
     max_open_connections: 0
     shared: true
     table_prefix: ""
-debug: true
-debug_profiling: true
-disable_itc: true
+debug: true # enable debug mode
+debug_profiling: true # enable endpoints that provides profiling information
+disable_itc: true # enable (ITC) incoming transaction checking
 # Prefixed with "_", because it's unused by default
 graphql:
-  enabled: false
-import_block_headers: ""
+  enabled: false # enables graphql
+import_block_headers: "" # url to import block headers
 # Prefixed with "_", because it's unused by default
 _new_relic:
-  domain_name: domain.com
-  enabled: false
-  license_key: BOGUS-LICENSE-KEY-1234567890987654321234
+  domain_name: domain.com # used for relic HostDisplayName
+  enabled: false # enable new relic
+  license_key: BOGUS-LICENSE-KEY-1234567890987654321234 # relic license key
 # Prefixed with "_", because it will be configured by default
 _nodes:
-  protocol: arc
-  apis:
+  protocol: arc # protocol - arc/mapi (arc is default)
+  apis: # list of apis used for getting and broadcasting transactions
     - arc_url: https://arc.gorillapool.io
       token:
-  use_fee_quotes: true
+  use_fee_quotes: true # use fee quotes for transaction fee calculation
   fee_unit:
     satoshis: 1
     bytes: 1000
 # Prefixed with "_", because it's unused by default
 _notifications:
```

Comment:
What kind of notification?

Reply:
There is a task BUX-499 about  investigation about it. We have some system where we can notify about new transactions and maybe it can be used for our bux-wallet